flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/connectors/hive/HiveDialectITCase.java

```diff
@@ -308,7 +308,7 @@ public void testAlterTable() throws Exception {
         // change location
         String newLocation = warehouse + "/tbl1_new_location";
         tableEnv.executeSql(
-                String.format("alter table `default`.tbl1 set location '%s'", newLocation));
+                String.format("alter table default.tbl1 set location '%s'", newLocation));
```

Comment:
Don't we support escape anymore?

Reply:
We do. This is to verify users no longer have to escape `default` keyword in hive dialect.

Reply:
Could you also add a test with escaption to make sure it also works?

```
`default`.tbl1
```